src/test/kotlin/org/rust/lang/core/completion/RsDeriveCompletionProviderTest.kt

```diff
@@ -37,7 +37,7 @@ class RsDeriveCompletionProviderTest : RsCompletionTestBase() {
     """)
 
     fun `test complete with dependencies`() {
-        StdDerivableTrait.values()
+        KnownDerivableTrait.values()
```

Comment:
This change brokes test because we can't find `Serialize` and `Fail` trait in index